helmclient.go

```diff
@@ -425,6 +413,49 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 	return nil
 }
 
+func (c *Client) ListReleaseContents() ([]*ReleaseContent, error) {
+	var resp *hapiservices.ListReleasesResponse
+	{
+		o := func() error {
+			t, err := c.newTunnel()
+			if IsTillerNotFound(err) {
+				return backoff.Permanent(microerror.Mask(err))
+			} else if err != nil {
+				return microerror.Mask(err)
+			}
+			defer c.closeTunnel(t)
+
+			resp, err = c.newHelmClientFromTunnel(t).ListReleases()
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			return nil
+		}
+		b := backoff.NewExponential(2*time.Minute, 60*time.Second)
+		n := func(err error, delay time.Duration) {
+			c.logger.Log("level", "debug", "message", "failed listing release contents")
+		}
```

Comment:
Backoff notifications and intervals code will also change in https://github.com/giantswarm/helmclient/pull/58. Let's make it right from the beggining.

Reply:
Done